pandas/core/indexes/multi.py

```diff
@@ -345,7 +345,7 @@ def set_levels(self, levels, level=None, inplace=False,
 
         Examples
         --------
-        >>> idx = MultiIndex.from_tuples([(1, u'one'), (1, u'two'),
+        >>> idx = pd.MultiIndex.from_tuples([(1, u'one'), (1, u'two'),
                                           (2, u'one'), (2, u'two')],
```

Comment:
so can you adjust formatting to line things up (in theory this would fail the doc-string tests if we were running them). do this generally in this PR (e.g. wheverever you are changing things)

Reply:
Yeah.. modifying them.. now..